mobify/progressive-web-scaffold
# iPhone 5 pdp coverage fix

iPhone 5 pdp coverage fix
- when using small devices such as iPhone 5 (width <= 320) t-plp__item-added-modal coverage is set to 60% in order to show all of its contents fully

 **JIRA**: https://mobify.atlassian.net/browse/HYB-983

## Changes
- on the container implemented a logic that checks for window size and sets the coverage to 40% if the device width is less than 320
- ideally use breakpoint to set the coverage % 

## Todo
- [x] review with someone who is familiar with scss 

## How to test-drive this PR
- run the simulator on iPhone 5
- select a product to buy and add it to cart
- on the slide up modal all of the contents should be delayed
